test/DbPostgreSQLAdapterTest/Tests/DbConnectionTest.cpp

```diff
@@ -0,0 +1,41 @@
+#include "stdafx.h"
+
+#include "Connection.h"
+#include "ConnectionConfiguration.h"
+#include "DbAdapterInterface/IDatabase.h"
+#include "Helpers/Helpers.h"
+#include "Helpers/DefaultConnectionConfiguration.h"
+
+
+using namespace testing;
+namespace systelab::db::postgresql::unit_test {
+
+	class DbConnectionTest : public Test
+	{
+		void SetUp() override
```

Comment:
Shouldn't be at least protected for TEST_F (that are derived classes) to use it?

Reply:
It's a very curious thing, it still works.. maybe the framework calls this method through a wrapper which is public or protected